deis-dev/tpl/objectstorage.toml

```diff
@@ -0,0 +1,46 @@
+#helm:generate helm template -o $HELM_GENERATE_DIR/manifests/deis-objectstorage-secret.yaml -d $HELM_GENERATE_FILE $HELM_GENERATE_DIR/tpl/deis-objectstorage-secret.yaml
+#
+# This is the main configuration file for Deis object storage. The values in
+# this file are passed into the appropriate services so that they can configure
+# themselves for persisting data in object storage.
+#
+# In general, all object storage credentials must be able to read and write to
+# the container or bucket they are configured to use.
+#
+# When you change values in this file, make sure to re-run `helm generate`
+# on this chart.
+
+# Set the storage backend
+#
+# Valid values are:
+# - filesystem: Store registry on ephemeral disk
+# - s3: Store registry in AWS S3 (configure in S3 section)
+# - azure: Store registry in Azure's object storage
+# - gcs: Store registry in Google Cloud Storage
+# - minio: Store registry on in-cluster Minio server
```

Comment:
> However, configuration is generic for other object storage users

Appearance of "registry" in the preceding lines might undermine that intent.  Can this be reworded somehow?